packages/flutter_tools/lib/src/flutter_plugins.dart

```diff
@@ -619,6 +629,65 @@ foreach(plugin ${FLUTTER_PLUGIN_LIST})
 endforeach(plugin)
 ''';
 
+const String _dartPluginRegisterWith = r'''
+      try {
+        {{dartClass}}.registerWith();
+      } catch (err) {
```

Comment:
This will also catch things like `TypeError`, `OutOfMemoryError` et cetera. I think in this case that might be what we want, but its generally against dart and flutter style guides so its worth commenting why this approach was taken.

Reply:
it now rethrows the exception, so we just had a hint to the stdout.